GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_Pioneer_NEW.cfg

```diff
@@ -0,0 +1,453 @@
+@PART[bluedog_Pioneer_ProbeCore]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.052
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Probe Core
+	@manufacturer = TRW
+	@description = Breakthroughs in materials and manufacturing have allowed engineers to design a super lightweight hexagonal satellite bus for cheap that can take exploration to new heights. Modular in design, this core is designed with slots to accommodate various expansions like control systems, electrical systems, thermal control systems, experiment bays, and more. <#ef7b06> AUTHOR'S NOTE: Add the Pioneer 10/11 Stellar Reference Assembly to your craft to gain additional SAS modes.</color>
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+	%MODULE[ModuleCommand]
+	{
+		%RESOURCE[ElectricCharge]
+		{
+			%rate = 0.1
+			///stated minimum (after turning off some instruments) is 80W, including comms
+		}
+	}
+	!MODULE[ModuleDataTransmitter]
+	{
+	}
+	!MODULE[ModuleReactionWheel]
+	{
+	}
+	!RESOURCE[ElectricCharge]
+	{
+	}
+	RESOURCE
+	{
+		name = MonoPropellant
+		amount = 0
+		maxAmount = 0
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 38.88
+		type = ServiceModule
+		basemass = -1
+		TANK
+		{
+			name = ElectricCharge
+			amount = 2880
+			maxAmount = 2880
+		}
+		TANK
+		{
+			name = Hydrazine
+			amount = 36
+			maxAmount = 36
+		}
+	}
+}
+
+
+@PART[bluedog_Pioneer_HGA]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.058
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 High Gain Antenna 
+	@manufacturer = TRW
+	@description = A large high gain dish antenna (and medium gain antenna) designed for the Pioneer 10/11 spacecraft that is very light for its size. Great at transmitting at vast distances at the cost of transmission speed. Includes integrated RCS thrusters and two sun sensors that gives a small amount of SAS control. Equipped with several aft-mounted micrometeorite detection panels. 
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+	@MODULE[ModuleDataTransmitter]
+    {
+        @antennaPower = 22500000000
+        @packetInterval = 0.3
+        @packetSize = 0.25
+        @packetResourceCost = 0.08
+    }

```

Comment:
Indentation

Reply:
Don't know what is wrong here, could oyu fix it for me or explain it more. :D

Reply:
![image](https://user-images.githubusercontent.com/25906300/171315953-c93bdc29-73e7-47af-9021-9fdf1e5429c2.png)
Tabs versus spaces issue.  In your editor you probably have tab length set up to match.  This is what it looks like in GItHub preview (click the "Files Changed" button).  Everything else is tabs, so just tabify this block.